ctapipe/image/waveform_cleaning.py

```diff
@@ -67,6 +67,26 @@ def apply(self, waveforms):
         return waveforms
 
 
+class BaselineWaveformCleaner(WaveformCleaner):
+    """
+    Basic waveform cleaner that subtracts the waveform baseline as
+    estimated  in a window of "baseline_width" samples, shifted of
+    "window_shift" samples
+    """
+    baseline_width = Int(10, help='Define then number of samples for estimating the '
```

Comment:
If `window_shift` is not equal to zero then it does not represents the `baseline_width` anymore right ?.
I would suggest to call the two parameters : `sample_start` `sample_end`